pkg/resources/secondary_connection.go

```diff
@@ -52,14 +52,15 @@ var secondaryConnectionSchema = map[string]*schema.Schema{
 
 func SecondaryConnection() *schema.Resource {
 	return &schema.Resource{
-		CreateContext: CreateSecondaryContextConnection,
-		ReadContext:   ReadSecondaryContextConnection,
-		UpdateContext: UpdateSecondaryContextConnection,
-		DeleteContext: DeleteSecondaryContextConnection,
-		Description:   "Resource used to manage secondary connections. For more information, check [connection documentation](https://docs.snowflake.com/en/sql-reference/sql/create-connection.html).",
+		CreateContext: CreateContextSecondaryConnection,
+		ReadContext:   ReadContextSecondaryConnection,
+		UpdateContext: UpdateContextSecondaryConnection,
+		DeleteContext: DeleteContextSecondaryConnection,
+		Description:   "Resource used to manage secondary connections. To promote secondary connection to primary check [migraton guide](https://github.com/Snowflake-Labs/terraform-provider-snowflake/blob/main/MIGRATION_GUIDE.md#connection-resources). For more information, check [connection documentation](https://docs.snowflake.com/en/sql-reference/sql/create-connection.html).",
```

Comment:
Hmm I'm not sure if linking to the migration guide is the best option here. Maybe we should just point to the primary resource here, and add a note in template/resources. The same for the other direction. WDYT?

Reply:
agree, description will be more clear, and migration of the resource will be in documentation